iquip/apps/explorer.py

```diff
@@ -96,6 +97,48 @@ def run(self):
         self.fetched.emit(experimentList, self.widget)
 
 
+class BuilderInfoDialog(QDialog):
+    """Dialog for setting builder info.
+    
+    There are two types of info.
+        cls: Target experiment class name.
+        tag: Additive tag for multiple builder apps.
+    """
+
+    def __init__(self, experimentInfos: List[str], parent: Optional[QWidget] = None):
```

Comment:
Why is the name `experimentInfos` instead of `clsNames` for example?

Reply:
Based on your usage, I think it is not `List` but `Iterable`.

Reply:
> Why is the name experimentInfos instead of clsNames for example?

Umm.. I also think `clsNames` is much proper. I will take it.